lib/material/material_ui.flow

```diff
@@ -839,7 +861,7 @@ MMultiAudioPlayer(filenames : [string], mStyle : [MMultiAudioPlayerStyle]) -> Ma
 							}
 							changeAudioPlaying(newIndex, getValue(playEx));
 
-							[
+							if (isSafari) [] else [
```

Comment:
Why do we skip this subscription? To prevent changing from play to paused icon visually? Or to prevent from missing initial playEx state?

How will be connection  isPlay -> playEx work for Safari if video stops for some reason from native video side (apart from it ends) ?

Reply:
> How will be connection isPlay -> playEx work for Safari

This connection sets playEx to false when we change isPlay[i]. When we pause started fragments, we always set playEx = false, so autoplay will never work in Safari.

We change the playEx value when we have completed all fragments in the playlist here. I think this is enough https://github.com/area9innovation/flow9/blob/master/lib/material/material_ui.flow#L715

Reply:
Video can be stopped by some other reason, but in this case it still be in 'playing' state on our level.

We can just save playEx initial value (the same way as volume) and use it in line 841

Reply:
> We can just save playEx initial value (the same way as volume)

No, this value may change after the player has been rendered. I'll rework this fconnect

Reply:
reworked. In Safari, these values will be connected after all fragments are initiated